assets/src/edit-story/elements/text/preview.js

```diff
@@ -57,8 +57,8 @@ function TextPreview( {
 		fontWeight: fontWeight ? fontWeight : null,
 		background: backgroundColor,
 		margin: 0,
-		color,
-		lineHeight,
+		color:color,
```

Comment:
I have no idea why our presubmit hook does this. Revert?

/cc @swissspidy

Reply:
I can't even revert without bypassing the husky tool:

```
$ gc -m 'Revert annoying autolint'
husky > pre-commit (node v12.12.0)
  ✔ Preparing...
  ✔ Running tasks...
  ✖ Applying modifications...
    → Prevented an empty git commit!
  ✔ Reverting to original state...
  ✔ Cleaning up...

  ⚠ lint-staged prevented an empty git commit.
    Use the --allow-empty option to continue, or check your task configuration

Prevented an empty git commit!
husky > pre-commit hook failed (add --no-verify to bypass)
```

It works with `--no-verify`.

Reply:
At this point I use `--no-verify` 100% of the time :(. TBH, Travis is sufficient protection for me. But we probably should fix it anyway.

Reply:
Bummer. There's probably two conflicting rules. That should be fixed by #156 though (separates formatting from rest of linting).

@dvoytenko Perhaps you can add your review on that PR too and we can merge it?